core/src/main/java/com/digitalpebble/stormcrawler/bolt/JSoupParserBolt.java

```diff
@@ -301,9 +300,7 @@ public void execute(Tuple tuple) {
             }
 
             Element body = jsoupDoc.body();
-            if (body != null) {
```

Comment:
what guarantee do you have that the _body_ returned is not null?

Reply:
The documentation of Document.body() states, that it'll create a body element if none is existing.

https://jsoup.org/apidocs/org/jsoup/nodes/Document.html#body()